Doc/library/turtle.rst

```diff
@@ -2778,7 +2778,7 @@ Changes since Python 3.0
   :func:`Screen.numinput <numinput>`. These pop up input dialogs and return
   strings and numbers respectively.
 
-- Two example scripts :file:`tdemo_nim.py` and :file:`tdemo_round_dance.py`
+- Two example scripts :file:`nim.py` and :file:`round_dance.py`
```

Comment:
I think this turtledemo item can be deleted.  Both examples are listed in the table above.

The 2.6 section should be deleted and I doubt this section is current.  'Added' notes should follow the item added, not collected together like this.  But these are turtle doc, not a turtledemo concern.

Reply:
The two lines were deleted.